.github/workflows/on-pull-request.yml

```diff
@@ -11,23 +12,34 @@ env:
     TEST_USER_IDENTIFIER: ${{ secrets.TEST_USER_IDENTIFIER }}
 
 jobs:
-    publish:
-        name: Run Tests on PR
-        runs-on: ubuntu-latest
-
-        steps:
-            - name: Checkout
-              uses: actions/checkout@v4
-
-            - name: Install Dependencies
-              run: npm i
-
-            - name: ESLint Check
-              run: npm run lint:ci
-
-            - name: Beachball Check
-              run: |
-                  npm run beachball-check
-
-            - name: Unit Test
-              run: npm run test
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
```

Comment:
Do the checkout actions need names?

Reply:
They can have names. With these actions the [default naming](https://github.com/passageidentity/passage-flex-node/actions/runs/11586674196/job/32257634819#step:2:1) is still pretty clear, so I opted to leave it as is